src/main/java/pl/allegro/tech/discovery/consul/recipes/watch/catalog/Services.java

```diff
@@ -27,6 +28,10 @@ public Set<String> serviceNames() {
         return serviceNamesToTags.keySet();
     }
 
+    public List<String> tagsForServiceOrNull(String serviceName) {
```

Comment:
po co dodatkowa metoda?

Reply:
Throwing exceptions is less performant, whereas `null` case is recoverable in my case :)